### PR TITLE
ci-operator: Move the operator-framework/operator-metering to kubernetes-reporting.

### PR DESCRIPTION
**Pull request by timflannagan**

### CODE REVIEW

**Comment by timflannagan1** [comment]
The jobs diff looks off.
/hold

File: ci-operator/jobs/kubernetes-reporting/metering-operator/kubernetes-reporting-metering-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
 presubmits:
-  operator-framework/operator-metering:
+  kubernetes-reporting/metering-operator:
   - agent: kubernetes
     always_run: true
     branches:
     - master
-    cluster: ci/api-build01-ci-devcluster-openshift-com:6443
+    cluster: api.ci
```
Comment (bparees) [inline ci-operator/jobs/kubernetes-reporting/metering-operator/kubernetes-reporting-metering-operator-master-presubmits.yaml:7]: not sure why this happened but i don't think we want this reverted.

**Pull request merged**